cfme/common/provider.py

```diff
@@ -707,6 +707,15 @@ def _prepare_endpoints(endpoints):
         else:
             raise ValueError("Endpoints should be either dict or endpoint class")
 
+    def get_console_connection_status(self):
```

Comment:
Add a comment explaining that these will only work if the console window is the active window.